process/block/interceptedBlocks/interceptedBlockHeader_test.go

```diff
@@ -61,6 +79,19 @@ func createMockShardHeader() *dataBlock.Header {
 	}
 }
 
+func createMockMarshallerWithHeaderV2Support() marshal.Marshalizer {
+	marshaler := &marshal.GogoProtoMarshalizer{}
+	m := &mock.MarshalizerStub{
```

Comment:
this is just a wrapper over the gogo proto marshalizer. Suggestion to return directly or better, just remove this function
```GO
func createMockMarshallerWithHeaderV2Support() marshal.Marshalizer {
    return &marshal.GogoProtoMarshalizer{}
}
```